flink-ai-flow/ai_flow/java/client/src/main/java/com/aiflow/client/MetadataClient.java

```diff
@@ -74,180 +74,164 @@ public MetadataClient(Channel channel) {
+    public DatasetMeta getDatasetById(Long datasetId) throws Exception {
+        IdRequest request = IdRequest.newBuilder().setId(datasetId).build();
+        Response response = metadataServiceStub.getDatasetById(request);
+        DatasetProto.Builder builder = DatasetProto.newBuilder();
+        return StringUtils.isEmpty(metadataDetailResponse(response, this.parser, builder)) ? null : buildDatasetMeta(builder.build());
     }
 
     /**
-     * Get a specific example in Metadata Store by example name.
+     * Get a specific dataset in Metadata Store by dataset name.
      *
-     * @param exampleName Name of example.
-     * @return Single ExampleMeta object if example exists, otherwise returns None if example does not exist.
+     * @param datasetName Name of dataset.
+     * @return Single DatasetMeta object if dataset exists, otherwise returns None if dataset does not exist.
      */
-    public ExampleMeta getExampleByName(String exampleName) throws Exception {
-        NameRequest request = NameRequest.newBuilder().setName(exampleName).build();
-        Response response = metadataServiceStub.getExampleByName(request);
-        ExampleProto.Builder builder = ExampleProto.newBuilder();
-        return StringUtils.isEmpty(metadataDetailResponse(response, this.parser, builder)) ? null : buildExampleMeta(builder.build());
+    public DatasetMeta getDatasetByName(String datasetName) throws Exception {
+        NameRequest request = NameRequest.newBuilder().setName(datasetName).build();
+        Response response = metadataServiceStub.getDatasetByName(request);
+        DatasetProto.Builder builder = DatasetProto.newBuilder();
+        return StringUtils.isEmpty(metadataDetailResponse(response, this.parser, builder)) ? null : buildDatasetMeta(builder.build());
     }
 
     /**
-     * Register a example in Metadata Store.
+     * Register a dataset in Metadata Store.
      *
-     * @param name        Name of example.
-     * @param supportType Example's support type.
-     * @param dataFormat  Data format of example.
-     * @param description Description of example.
-     * @param batchUri    Batch uri of example.
-     * @param streamUri   Stream uri of example.
-     * @param createTime  Time when example is created.
-     * @param updateTime  Time when example is updated.
-     * @param properties  Properties of example.
-     * @param nameList    Name list of example's schema.
-     * @param typeList    Type list corresponded to name list of example's schema.
-     * @return Single ExampleMeta object registered in Metadata Store.
+     * @param name        Name of dataset.
+     * @param dataFormat  Data format of dataset.
+     * @param description Description of dataset.
+     * @param uri    Uri of dataset.
+     * @param createTime  Time when dataset is created.
+     * @param updateTime  Time when dataset is updated.
```

Comment:
`createTime` and `updateTime` don't need to define as parameters for `registerDataset`.